README.md

```diff
@@ -1 +1,60 @@
 Web front-end for the European Variation Archive (EVA), developed using technologies such as JavaScript and HTML5.
+
+To install EVA -Web app do the following steps.
```

Comment:
Thanks for adding this! For consistency with other team repositories, please format the document following the same style as https://github.com/EBIvariation/eva-pipeline